cordova-lib/src/plugman/browserify.js

```diff
@@ -157,8 +157,8 @@ module.exports = function doBrowserify (project, platformApi, pluginInfoProvider
         // instead of generating intermediate file on FS
         var cordova_plugins = new Readable();
         cordova_plugins.push(
-            'module.exports.metadata = ' + JSON.stringify(pluginMetadata, null, 4) + ';\n' +
-            'module.exports = ' + JSON.stringify(modulesMetadata, null, 4) + ';\n', 'utf8');
+            'module.exports = ' + JSON.stringify(modulesMetadata, null, 4) + ';\n' +
+            'module.exports.metadata = ' + JSON.stringify(pluginMetadata, null, 4) + ';\n', 'utf8');
```

Comment:
Right. And the behavior differs for normal and browserify cases.
I.e. in normal case when we are injecting a script element the above code did not result in `module.exports` being overwritten.